src/admin/assignments/assignments.types.ts

```diff
@@ -25,6 +25,7 @@ export interface AssignmentsOverviewTableState extends FilterableTableState {
 	share_type: string;
 	subjects: string[];
 	education_levels: string[];
+	education_level: string[]; // "Kenmerk"
```

Comment:
maybe add a bit more comments here
```suggestion
	education_levels: string[]; // These are the lom values for the assignment publication details
	education_level: string[]; // This is the type of assignment: Lager onderwijs or secundair onderwijs. also known as "Kenmerk"
```